docs/hub/enterprise-datasets.md

```diff
@@ -0,0 +1,9 @@
+# Datasets
+
+<Tip warning={true}>
+This feature is part of the <a href="https://huggingface.co/enterprise" target="_blank">Enterprise Hub</a>.
+</Tip>
+
+The Dataset Viewer is enabled on private datasets owned by an Enterprise organization. The Dataset Viewer allows teams to understand their data and to help them build better data processing and filtering for AI. The Viewer allows to explore the datasets content, inspect data distributions, filter by values and even search for keywords. It also includes the datasets conversion to Parquet which can be used for programmatic data visualization.
```

Comment:
```suggestion
The Dataset Viewer is enabled on private datasets owned by an Enterprise Hub organization. 

The Dataset Viewer allows teams to understand their data and to help them build better data processing and filtering for AI. The Viewer allows to explore the datasets content, inspect data distributions, filter by values and even search for keywords. It also includes the datasets conversion to Parquet which can be used for programmatic data visualization.
```

(the product name is Enterprise Hub not Enterprise)

maybe add a visual screenshot too?

Reply:
done